### PR TITLE
:recycle: Rework answers validation

### LINKED ISSUE #2: Add explicit model for valid form and error handling
Currently, our code transforms a big payload of Typeform webhook to small easy-to-read `dict`. But we don't have an explicit model of valid form:
https://github.com/piterpy-meetup/c4p2n/blob/6da7d525c84f3f90e3cf4e5224c068fa75585a6b/c4p2n/models.py#L94-L99

It can cause errors on this level:
https://github.com/piterpy-meetup/c4p2n/blob/6da7d525c84f3f90e3cf4e5224c068fa75585a6b/c4p2n/notion.py#L14-L25

### PR DESCRIPTION
**Pull request by ave2me**
Closes #2.

### CODE REVIEW

**Comment by ave2me** [comment]
Removed `extractors.py` and `constants.py` modules as they became obsolete due to models redesign. Also added basic error handling in `call_for_paper_webhook` endpoint.

File: c4p2n/models.py [hunk 1]
```diff
@@ -62,6 +74,26 @@ class CallForPaperFormResponse(BaseModel):
     answers: List[Union[TextAnswer, MultipleChoiceAnswer]]
 
 
+class CallForPaperAnswers(BaseModel):
```
Comment (b0g3r) [inline c4p2n/models.py]: What about renaming to `CallForPaperApplication` or something without plural `answers`

File: c4p2n/models.py [hunk 2]
```diff
@@ -62,6 +74,26 @@ class CallForPaperFormResponse(BaseModel):
...
+    photo_link: str
+    talk_date: List[str]
+    telegram: str = Field(default="")
+    contact: str = Field(default="")
+    phone: str
+    talk_title: str = Field(default="")
+    talk_description: str
+    questions: str = Field(default="")
+
+    def talk_dates(self) -> str:
```
Comment (b0g3r) [inline c4p2n/models.py:90]: I'm not sure why talk_dates isn't validator for talk_date? Maybe property? Or rewrite `MultipleChoiceAnswer.extract` to `-> str`

**Review by b0g3r — CHANGES_REQUESTED** [review]

File: c4p2n/models.py (hunk 2 above)
Comment (ave2me) [reply c4p2n/models.py:90]: Started using property.

**Review by b0g3r — APPROVED** [review]

**Pull request merged**